std/format.d

```diff
@@ -5777,7 +5777,16 @@ private bool needToSwapEndianess(Char)(const ref FormatSpec!Char f)
     assert(r == ">            15<, [1, 2, 3]");
 
     assert(format("%8s", "bar") == "     bar");
-    assert(format("%8s", "b\u00e9ll\u00f4") == " b\u00e9ll\u00f4");
+    assert(format("%8s", "b\u00e9ll\u00f4") == "   b\u00e9ll\u00f4");
+}
+
+@safe pure unittest // bugzilla 18205
+{
+    assert("|%8s|".format("abc")        == "|     abc|");
+    assert("|%8s|".format("αβγ")        == "|     αβγ|");
+    assert("|%8s|".format("   ")        == "|        |");
+    assert("|%8s|".format("été")        == "|     été|");
+    assert("|%8s|".format("été 2018")   == "|été 2018|");
```

Comment:
Just a thought: since Unicode contains precomposed characters for `é`, etc., it might be better to spell out the grapheme clustering explicitly, e.g., `"e\u0301"`, so that it's clear what the unittest is checking.